lighthouse-core/audits/byte-efficiency/byte-efficiency-audit.js

```diff
@@ -34,7 +35,7 @@ class UnusedBytes extends Audit {
    * @return {string}
    */
   static bytesToKbString(bytes) {
-    return Math.round(bytes / KB_IN_BYTES).toLocaleString() + ' KB';
+    return `${Util.formateBytesToKB(bytes)} KB`;
```

Comment:
Ok, I'll add the 'KB' inside this helper, similar how it's done now for milliseconds. One question though as `formatMilliseconds` helper looks:
```
 return `${coarseTime.toLocaleString()}${NBSP}ms`;
```
Which (at least on my machine) renders to sth like 1,000ms so there is no space between number and `ms`. Should I done it the same way or just ad ` KB` at the end for `formatBytestoKB`?